src/tests/functional/plugin/shared/src/behavior/plugin/configuration_tests.cpp

```diff
@@ -188,7 +200,9 @@ TEST_P(DefaultValuesConfigTests, CanSetDefaultValueBackToPlugin) {
 }
 
 TEST_P(ExclusiveAsyncReqTests, excluAsyncReqTests) {
-    ASSERT_NO_THROW(ie->SetConfig(configuration, target_device));
+    if (target_device.find("AUTO") == std::string::npos && target_device.find("MULTI") == std::string::npos) {
+        ASSERT_NO_THROW(ie->SetConfig(configuration, target_device));
+    }
```

Comment:
why do these tests worked before?

Reply:
1) auto supported keys only, like enable_startup_fallback
2) core keys, like batching/caching_dir, these should be removed from auto support list, and not allowed to be set by set_property  (cache_dir has been removed, and batching is in progress)
3) some generic hardware keys, like exclusive_async_request/execution mode, are also treated as auto supported keys, this is used by underlying hardware plugins, as I explained before, we can set to auto plugin by core.set_property("AUTO", exclusive***), and then user don't need to explicitly add this property for each compile model.
So I think the main issue we need to figure out @WeldonWangwang  @wangleis  is for case 3), is this key really matters from performance perspective(switch on off significantly affect GPU perf or some other important inference metrics) and we can get benefit by setting it only once in AUTO, if not, we should totally wipe out the case 3) as well.

Reply:
Ok, I missed the part that tests are related to SetConfig. Here I agree that auto should not support these properties. 
For 3 case users can use:
```
core.compile_model(AUTO, model, ov::device::properties(GPU, exclusive_async_requests(true)));
```

Reply:
Okay. Thank you all for the very clear explanation.

Reply:
should we remove this test instantiate for auto/multi instead of modify the test_p here?

Reply:
This test instantiate just for AUTO test, I think it can be remove now.

Reply:
agree to remove